src/CMakeLists.txt

```diff
@@ -27,7 +27,14 @@ if (WIN32)
 endif()
 
 if(INPUTLEAP_BUILD_TESTS)
-    include(../cmake/gtest.cmake)
+    include(FetchContent)
+    FetchContent_Declare(
+        googletest
+        GIT_REPOSITORY https://github.com/google/googletest.git
+        GIT_TAG        b514bdc898e2951020cbdca1304b75f5950d1f59 # release-1.15.2
+        FIND_PACKAGE_ARGS NAMES GTest
+    )
+    FetchContent_MakeAvailable(googletest)
```

Comment:
```suggestion
    set(INSTALL_GTEST OFF)
    FetchContent_MakeAvailable(googletest)
```

Reply:
What are you trying to suggest here?

Reply:
What I explained above, to add `set(INSTALL_GTEST OFF)` before making googletest available so it doesn’t install `gmock` and its other targets (and thus no need to be removing them…)

Reply:
This will happen when FetchContent_MakeAvailiable is called.  This is documented in the [cmake docs](https://cmake.org/cmake/help/latest/module/FetchContent.html#integrating-with-find-package)

Reply:
This stems from the initial attempt of the PR author to do this without the Build Depends and then to make sure if you didn't have them and they got installed by Fetch_Content it was removed so the rpm build didn't fail. TLDR they need to put the build depends part back so we use in repo version of gtest / gmock

Reply:
I think it's right to set INSTALL_GTEST OFF (see https://github.com/google/googletest/blob/main/CMakeLists.txt)
I changed that and removed the newest commit, the Fedora spec file is now back as it was.